pylintrc

```diff
@@ -166,6 +166,7 @@ disable=print-statement,
         not-an-iterable,
         unsubscriptable-object,
         unsupported-assignment-operation,
+        attribute-defined-outside-init,
```

Comment:
Why do we need to disable this?

Reply:
We shouldn't need to. We set instance variables outside of the constructor in many places throughout MLflow and lint seems to pass.